simulator-ui/src/main/resources/static/src/app/services/activity-service.ts

```diff
@@ -11,22 +12,18 @@ export class ActivityService {
 
     private serviceUrl = 'api/activity';
 
-    getScenarioExecutions(page: number, pageSize: number): Observable<ScenarioExecution[]> {
-        return this.http.get<ScenarioExecution[]>(this.serviceUrl + "?page=" + page + "&size=" + pageSize);
+    getScenarioExecutions(filter: ScenarioExecutionFilter): Observable<ScenarioExecution[]> {
+        return this.http.post<ScenarioExecution[]>(
+            this.serviceUrl,
+            JSON.stringify(filter),
```

Comment:
I think you can post the JSON "as is" (without `JSON.stringify`).

Reply:
is implemented like this in all other services.. maybe subject for a later change.